homeassistant/components/plum_lightpad/light.py

```diff
@@ -64,6 +94,16 @@ def name(self):
         """Return the name of the switch if any."""
         return self._load.name
 
+    @property
+    def device_info(self):
+        """Return the device info."""
+        return {
+            "name": self.name,
+            "identifiers": {(DOMAIN, self.unique_id)},
+            "model": "Dimmer",
+            "manufacturer": "Plum",
+        }
+
     @property
     def brightness(self) -> int:
```

Comment:
The load is physically attached to one of the lightpad dimmer switches - but there can be multiple lightpads associated with a LogicalLoad.  Something about calling the model "Dimmer" here doesn't sit well.

Reply:
What does it look like when there is one logical load and multiple lighpads? Is there a real world Plum device like this? Can you point me to any docs you have? The way I see it that there is one 'logical' device which has two 'physical' devices inside it, a dimmer and a glow ring. Am I wrong?

Reply:
This is inside the PlumLight class which, IMO, represents a HA device for controlling light, in this with dimming support. So, I though "Dimmer" is appropriate, as Plum API does not report true "model". But if you disagree - what do you want to call it? I can leave it Nil or use your judgment call. LMK

Reply:
Yeah, I'm OK with it for now.  I don't have a better name at the moment, so leave it for now

Reply:
Yeah, I'm OK with it for now. I don't have a better name at the moment, so leave it for now